woocommerce/woocommerce
# Remove onboarding welcome modals

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





The welcome modals are already disabled in https://github.com/woocommerce/woocommerce/pull/32331 and we've decided to show confetti (https://github.com/woocommerce/woocommerce/issues/32159) instead of the welcome modals. This PR removes the welcome modals from the codebase so we don't fetch the related data from the server and reduce the code complexity.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:



1. Review the code changes and ensure that the welcome modals are removed from the codebase.
2. Create a fresh site 
3. Complete the core profiler
4. Go to WooCommerce > Home and confirm it loads properly without any errors



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
